src/main/java/org/sonar/squidbridge/annotations/SqaleSubCharacteristic.java

```diff
@@ -19,20 +19,20 @@
  */
 package org.sonar.squidbridge.annotations;
 
-import org.sonar.api.server.rule.RulesDefinition.SubCharacteristics;
-
 import java.lang.annotation.ElementType;
 import java.lang.annotation.Retention;
 import java.lang.annotation.RetentionPolicy;
 import java.lang.annotation.Target;
 
+/**
+ * @deprecated. Will be removed without alternative.
+ * Since SQ 5.6 LTS, SQALE is not used in our model and sub characteristics are ignored.
+ */
+@Deprecated
 @Retention(RetentionPolicy.RUNTIME)
 @Target(ElementType.TYPE)
 public @interface SqaleSubCharacteristic {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Do not forget to remove this deprecated code someday. ([squid:S1133](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1133))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.sslr-squid-bridge%3Asslr-squid-bridge&branch=SSLRSQBR-71&issues=AWHXxlp2af12pdu-6jFB&open=AWHXxlp2af12pdu-6jFB)

[ISSUE_KEY]: AWHXxlp2af12pdu-6jFB